browser/resources/settings/email_aliases_page/tests/email_aliases_modal.test.tsx

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { render, screen, waitFor, act, fireEvent } from '@testing-library/react'
```

Comment:
nit: `fireEvent` is unused.

`eslint` doesn't seem to run properly on Jest

Reply:
fixed